pkg/setting/setting_plugins.go

```diff
@@ -78,3 +55,18 @@ func (cfg *Cfg) readPluginSettings(iniFile *ini.File) error {
 
 	return nil
 }
+
+// readPluginIDsList takes a config value containing a list of plugin ids separated by commas and returns the
+// corresponding slice of strings. Each value has its leading and trailing whitespace removed. Empty values are
+// discarded.
+func readPluginIDsList(configValue string) []string {
```

Comment:
FYI the below util function is used throughout the setting package so we should maybe reuse that for consistency?

https://github.com/grafana/grafana/blob/106903b549d8744597144cc5151fd0b08b210973/pkg/util/strings.go#L32-L48

Reply:
Thanks, I was not aware of this! I have updated it so it uses `util.SplitString` instead